wiki/Tournaments/OWC/2022/en.md

```diff
@@ -317,7 +318,7 @@ Saturday, 12 November 2022:
 | Team A |  |  | Team B | Match link |
 | --: | :-: | :-: | :-- | :-- |
 | **Hong Kong** ::{ flag=HK }:: | **6** | 1 | ::{ flag=ID }:: Indonesia | [#1](https://osu.ppy.sh/community/matches/105012634) |
-| Romania ::{ flag=RO }:: | 3 | **6** | ::{ flag=CN }:: **China** | [#1](https://osu.ppy.sh/community/matches/105014920) |
+| **Romania** ::{ flag=RO }:: | **6** | FF | ::{ flag=CN }:: China | [1# \(WBD\)](https://osu.ppy.sh/community/matches/105014920) |
```

Comment:
we have standard formatting for these (and just flipping the numbers around doesn't make sense)

```suggestion
| **Romania** ::{ flag=RO }:: | **0** | -1 | ::{ flag=CN }:: China | *win by default* |
```